templates/docs/examples/base/forms/datalist.html

```diff
@@ -0,0 +1,19 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Forms / Datalist{% endblock %}
+
+{% block standalone_css %}base{% endblock %}
+
+{% block content %}
+<form>
+  <label for="ice-cream-choice">Choose a flavor:</label>
+  <input list="ice-cream-flavors" id="ice-cream-choice" name="ice-cream-choice" />
```

Comment:
For the input to be styled properly in Vanilla it needs type:

```suggestion
  <input type="text" list="ice-cream-flavors" id="ice-cream-choice" name="ice-cream-choice" />
```